crates/sncast/src/starknet_commands/account/add.rs

```diff
@@ -92,21 +91,21 @@ pub async fn add(
     let chain_id = get_chain_id(provider).await?;
     write_account_to_accounts_file(account, accounts_file, chain_id, account_json.clone())?;
 
-    if add.add_profile {
+    if add.add_profile.is_some() {
         let config = CastConfig {
             rpc_url: rpc_url.into(),
             account: account.into(),
             accounts_file: accounts_file.into(),
             ..Default::default()
         };
-        add_created_profile_to_configuration(path_to_scarb_toml, &config)?;
+        add_created_profile_to_configuration(&add.add_profile, &config, &None)?;
     }
 
     Ok(AccountAddResponse {
-        add_profile: if add.add_profile {
-            "Profile successfully added to Scarb.toml".to_string()
+        add_profile: if add.add_profile.is_some() {
+            "Profile successfully added to snfoundry.toml".to_string()
```

Comment:
nit: It would be nice to include a name of the created profile here.

Reply:
added https://github.com/foundry-rs/starknet-foundry/pull/1644/commits/af08cbf36a17154306ea6f8cad673856ebb4cb20